docs/tutorials/basics/50_control_flow.md

```diff
@@ -73,7 +73,7 @@ p! "foo" && nil,
    "bar" || "foo"
 ```
 
-The `NOT`, `XOR` and equivalence operators always return a `Bool` value (`true` or `false`).
+The `NOT`, `XOR`, and equivalence operators always return a `Bool` value (`true` or `false`).
```

Comment:
I am against this `, and` business. I think most people prefer to never use a comma before "and".

Reply:
In this case, it's not strictly necessary, true. But in others, it helps avoid ambiguity.

Reply:
Not that it's relevant in any way, but the reason it's particularly jarring to me is that in Ukrainian language `and` means `and`, while `, and` means `and then`.

Don't see how it prevents ambiguity.

Either way, not gonna push this one.

Reply:
https://en.wikipedia.org/wiki/Serial_comma#Ambiguity